cli/cmd/vuln_host_show_assessment.go

```diff
@@ -58,6 +58,22 @@ Grab a CVE id and feed it to the command:
 					vulCmdState.Details = true
 				}
 			}
+
+			// validate collector_type flag
+			if vulCmdState.CollectorType != vulnHostCollectorTypeAgentless && vulCmdState.CollectorType != vulnHostCollectorTypeAgent {
+				return errors.Errorf("collector_type must be either %s or %s", vulnHostCollectorTypeAgent, vulnHostCollectorTypeAgentless)
+			}
```

Comment:
nit; could be a switch statement to avoid nested ifs 
```suggestion
			switch vulCmdState.CollectorType {
			case vulnHostCollectorTypeAgentless, vulnHostCollectorTypeAgent:
			default:
				return errors.Errorf(
					"collector_type must be either %s or %s",
					vulnHostCollectorTypeAgent, vulnHostCollectorTypeAgentless,
				)
			}
```

Reply:
There is no nested if on the line with this suggestion.